OverlayPlugin.Core/PluginMain.cs

```diff
@@ -232,120 +212,87 @@ public void InitPlugin(TabPage pluginScreenSpace, Label pluginStatusText)
                     _logger.Log(LogLevel.Error, string.Format("Failed to load presets: {0}", ex));
                 }
 
-                this.label.Text = "Init Phase 1: Waiting for plugins to load";
-                initTimer = new Timer();
-                initTimer.Interval = 300;
-                initTimer.Tick += async (o, e) =>
+                await SetPluginStatusText("Init Phase 1: Waiting for plugins to load");
+
+                ActGlobals.oFormActMain.VisibleChanged += Handle_FormActMain_VisibleChanged;
```

Comment:
Unless I missed something, the plugin init can get stuck here since this event is never triggered if you enable OverlayPlugin after launching ACT.

Reply:
I believe that 4986de7c889c9f9c83b7e09e261438c183df7af8 should resolve this?